```
error: builder for '/nix/store/lqhs7bwr33fipyk3iyarwyg10fl2383n-libredwg-0.12.4.drv' failed with exit code 2;
       last 10 log lines:
       >         ^
       > 1 warning generated.
       >   CCLD     llvmfuzz_standalone
       > ld: file not found: /nix/store/j8w19vhk1yazy96zm22apalqm1jvvlrg-clang-wrapper-11.1.0/resource-root/lib/darwin/libclang_rt.asan_osx_dynamic.dylib
       > clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
       > make[2]: *** [Makefile:763: llvmfuzz_standalone] Error 1
       > make[2]: Leaving directory '/private/tmp/nix-build-libredwg-0.12.4.drv-0/source/examples'
       > make[1]: *** [Makefile:676: all-recursive] Error 1
       > make[1]: Leaving directory '/private/tmp/nix-build-libredwg-0.12.4.drv-0/source'
       > make: *** [Makefile:559: all] Error 2
       For full logs, run 'nix log /nix/store/lqhs7bwr33fipyk3iyarwyg10fl2383n-libredwg-0.12.4.drv'.
```

[Full log](https://gist.github.com/mikroskeem/61960ffd4553f294fe91a2e61f4908bd)

--- a/pkgs/development/libraries/libredwg/default.nix
+++ b/pkgs/development/libraries/libredwg/default.nix
@@ -34,7 +34,7 @@ stdenv.mkDerivation rec {
   checkInputs = lib.optionals enablePython [ libxml2 libxml2.dev ];
 
   meta = with lib; {
-    broken = stdenv.isDarwin;
+    broken = stdenv.isDarwin && stdenv.isAarch64;
     description = "Free implementation of the DWG file format";
     homepage = "https://savannah.gnu.org/projects/libredwg/";
     maintainers = with maintainers; [ tweber ];

